### PR TITLE
Highlight major changes on the changelog overlay

### LINKED ISSUE #4942: Allow highlighting changes in changelog
Highlighting changes isn't currently available in Changelog Listing, as it is on the website

 web

 lazer

### PR DESCRIPTION
**Pull request by Game4all**
Resolves #4942 
Major changes in changelog entries are now highlighted on the changelog overlay (with the same color as on web)
image

### CODE REVIEW

File: osu.Game/Overlays/Changelog/ChangelogBuild.cs [hunk 1]
```diff
@@ -69,34 +69,69 @@ public ChangelogBuild(APIChangelogBuild build)
                         Margin = new MarginPadding { Vertical = 5 },
                     };
 
+                    var entryColor = entry.Major != null && (bool)entry.Major ? OsuColour.FromHex("#fd5") : Color4.White;
```
Comment (peppy) [inline osu.Game/Overlays/Changelog/ChangelogBuild.cs]: please use the colour from the palette, not hardcoded locally.

Comment (HoutarouOreki) [reply osu.Game/Overlays/Changelog/ChangelogBuild.cs]: the variable could be renamed to `entryColour` to stay consistent with the language

File: osu.Game/Overlays/Changelog/ChangelogBuild.cs [hunk 2]
```diff
@@ -69,34 +74,69 @@ public ChangelogBuild(APIChangelogBuild build)
                         Margin = new MarginPadding { Vertical = 5 },
                     };
```
Comment (peppy) [inline osu.Game/Overlays/Changelog/ChangelogBuild.cs]: this shouldn't be nullable i don't think

**Comment by peppy** [comment]
Thanks for the contribution!

**Review by peppy — APPROVED** [review]

**Pull request merged**